README.md

```diff
@@ -4,7 +4,9 @@
 ![rubocop workflow](https://github.com/belgamo/comarev/actions/workflows/rubocop.yml/badge.svg)
 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+
```

Comment:
I think it's better not to change this section. It's used to make the @allcontributors bot work, can you please remove it? 😄

Reply:
Sure! This change was not intended